index.js

```diff
@@ -7,6 +7,6 @@ const batchSize = 500;
 const batchTime = 5000;
 
 const serverLogger = new ServerLogger(batchSize, batchTime);
-export function createClient(appId) {
-	return new LoggingClient(appId, serverLogger);
+export function createClient(appId, shouldThrottle) {
+	return new LoggingClient(appId, shouldThrottle, serverLogger);
```

Comment:
Nothing else uses `LoggingClient` directly, so this addition doesn't break anything

Reply:
Con-eval uses it now

Reply:
> Con-eval uses it now

misspoke, we're using `createClient`

Reply:
Yeah, it wouldn't break what you have, you'll just get throttling off because you aren't passing in a second parameter.  I like the idea of changing this to an optional object parameter though, now that it sounds like we might want to make the throttle time configurable